neural_compressor/torch/algorithms/weight_only/gptq.py

```diff
@@ -850,16 +849,16 @@ def find_params(self, x, weight=False):
             self.zero = self.zero.reshape(shape)
 
             if self.double_quant:
-                from .rtn import quant_weight
+                from .utility import quant_tensor
 
                 orig_scale_shape = self.scale.shape
                 self.scale = self.scale.reshape(1, -1)
-                self.scale = quant_weight(
+                quant_tensor(

```

Comment:
No returns, `self.scale` should be updated.

Reply:
The `quant_tensor` is an in-place ops now to save memory, so later we should remove the return. Please aware that scale and zp are still required to return when the return contains them.